plugins/cassandra/cassandraembed.hpp

```diff
@@ -469,7 +475,7 @@ class CassandraStatementInfo : public CInterface
 extern bool getBooleanResult(const RtlFieldInfo *field, const CassValue *value);
 extern void getDataResult(const RtlFieldInfo *field, const CassValue *value, size32_t &chars, void * &result);
 extern __int64 getSignedResult(const RtlFieldInfo *field, const CassValue *value);
-extern unsigned __int64 getUnsignedResult(const RtlFieldInfo *field, const CassValue *value);
+extern CASSANDRAEMBED_API unsigned __int64 getUnsignedResult(const RtlFieldInfo *field, const CassValue *value);
```

Comment:
We should probably put CASSANDRAEMBED_API on all these externs

Reply:
OK. How about class CassandraPrepared, CassandraFutureResult, CassandraBatch, CassandraStatement, CassandraRetryingFuture, CassandraResult, CassandraIterator, CassandraCollection, and CassandraStatementInfo?

Reply:
May as well.